File: javalin5/javalin-testing-example/src/main/java/io/javalin/example/java/Application.java
package io.javalin.example.java;

import io.javalin.Javalin;
import static io.javalin.apibuilder.ApiBuilder.get;
import static io.javalin.apibuilder.ApiBuilder.post;

public class Application {

    private Javalin app;

    public Application(String dependency) {
        app = Javalin.create().routes(() -> {
            get("/users", UserController::getAll);
            post("/users", UserController::create);
            get("/ui", ctx -> ctx.html("<h1>User UI</h1>"));
        });
    }

    public Javalin javalinApp() {
        return app;
    }

}

File: javalin5/javalin-testing-example/src/main/kotlin/io/javalin/example/kotlin/Application.kt
package io.javalin.example.kotlin

import io.javalin.Javalin
import io.javalin.apibuilder.ApiBuilder.get
import io.javalin.apibuilder.ApiBuilder.post

class Application(dependency: String) {

    val app = Javalin.create().routes {
        get("/users", UserController::getAll)
        post("/users", UserController::create)
        get("/ui") { it.html("<h1>User UI</h1>") }
    }

}


